pkg/platform/aws/template.go

```diff
@@ -198,6 +198,8 @@ module "worker-pool-{{ $index }}" {
   clc_snippets          = {{ (index $.WorkerpoolCfg $index "clc_snippets") }}
   {{- end }}
 
+  cpu_manager_policy = "{{$pool.CPUManagerPolicy}}"
```

Comment:
Terraform variable description says `Possible values: `none`, `static`.`, while this will put `""`, which IIRC is different from `null` or not specifying it, so it can pick up the default value.

Reply:
Yep "" is put in here, which is equally treated as empty. And the default value of `non` is picked up.